.github/workflows/spread-large.yaml

```diff
@@ -0,0 +1,50 @@
+name: Large Manual Spread
+on:
+  schedule:
+    # At 03:00 on Wednesday and Sunday.
+    - cron: "0 3 * * WED,SUN"
```

Comment:
From my research, the last person to enable a scheduled action will receive email notifications when it fails.  So you will get email notifications on failure.

I think this is OK for now.  In the future, perhaps we can create a better notification process.  We could use a github action to create an issue, email all maintainers, or create a mattermost post.